tools/ARIAtools/extractProduct.py

```diff
@@ -474,6 +511,48 @@ def merged_productbbox(
     else:
         bbox_file = prods_TOTbbox
 
+    # Compare current bbox to existing bbox
+    if exist_bbox and run_log:
+        exist_area = ARIAtools.util.shp.shp_area(exist_bbox, lyr_proj)
+
+        # Calculate overlap area
+        new_bbox = ARIAtools.util.shp.open_shp(prods_TOTbbox)
+        new_area = ARIAtools.util.shp.shp_area(new_bbox, lyr_proj)
+        area_ratio = new_area / exist_area
+
+        olap_bbox = exist_bbox.intersection(new_bbox)
+        olap_area = ARIAtools.util.shp.shp_area(olap_bbox, lyr_proj)
+
+        # Compare areas
+        delta_area = np.abs(olap_area - exist_area)
+        delta_area = np.round(delta_area * 1E7) * 1E-7
+
+        olap_ratio = olap_area / exist_area
+        olap_ratio = np.round(olap_ratio * 1E7) * 1E-7
+
+        if verbose:
```

Comment:
agree with Alex to replace verbose with logger

Reply:
Agreed. Implemented in https://github.com/aria-tools/ARIA-tools/pull/456/commits/00ffde6bdd37aaa2feab53d8393e4d76831d2eb9